h2/src/main/org/h2/command/Parser.java

```diff
@@ -1572,6 +1572,9 @@ private Prepared parseShow() {
         } else if (readIf("DATESTYLE")) {
             // for PostgreSQL compatibility
             buff.append("'ISO' DATESTYLE");
+        } else if (readIf("SEARCH_PATH")) {
+            // for PostgreSQL compatibility
+            buff.append('\'').append(String.join(", ", session.getSchemaSearchPath())).append('\'');
```

Comment:
1. You need to double `'` characters here. Use `StringUtils.quoteStringSQL(StringBuilder, String)` for it.

2. Some schema names need to be enclosed into `"` characters. Take a look on `Parser.quoteIdentifier()`. If you don't want to quote all of them for a some reason, you can't use this method directly, but you can do something similar with real `databaseToUpper` and `databaseToLower` flags passed to `ParserUtil.isSimpleIdentifier()`.

Reply:
`session.getSchemaSearchPath()` may return `null` and this situation should be handled too.